src/Admin/BlockAdmin.php

```diff
@@ -14,10 +14,14 @@
 namespace Sonata\PageBundle\Admin;
 
 use Doctrine\ORM\EntityRepository;
+use InvalidArgumentException;
```

Comment:
Personally, I prefer to avoid the import for classes without NS (at least for native classes, see FriendsOfPHP/PHP-CS-Fixer#2166).

Reply:
AFAIK we don't have a prefered code style for this and use both kinds.

Reply:
Yes, but IMO importing native types leads to confusion when there are domain classes with the same names.

Reply:
This one is new as far as I see, so this should be located in the block bundle exception namespace

Reply:
Changed this to a `RuntimeException` as this error should never happen